data_sender.py

```diff
@@ -1,20 +1,21 @@
-import math
 import argparse
-import numpy as np
 import serial
+from ball_finder import BallFinder
+from depth_live_generator import DepthLiveGenerator
 from pose_calculator import PoseCalculator
+from power_cell_detector import PowerCellDetector
 from power_port_detector import PowerPortDetector
 from video_live_generator import VideoLiveGenerator
 
 
 class DataSender:
-
-    def __init__(self, name="ttyS0", rate=9600, port=0):
+    def __init__(self, name="ttyS0", rate=9600, video_port=0, depth_port=1):
```

Comment:
We need to have an option to run without the depth data, so when this code is merged onto master we can still test the power port detection independently.